appservice/src/SiteWrapper.ts

```diff
@@ -202,7 +216,13 @@ export class SiteWrapper {
                 break;
         }
 
-        outputChannel.appendLine(localize('deployComplete', '>>>>>> Deployment to "{0}" completed. <<<<<<', this.appName));
+        const displayUrl: string = this.isFunctionApp ? '' : this.defaultHostName;
+        outputChannel.appendLine(
+            displayUrl ?
```

Comment:
nit: displayUrl doesn't seem necessary. You could just do something like:
```
!this.isFunctionApp ?
    localize('deployCompleteWithUrl', '>>>>>> Deployment to "{0}" completed: {1} <<<<<<', this.appName, `https://${this.defaultHostName}`) :
    localize('deployComplete', '>>>>>> Deployment to "{0}" completed. <<<<<<', this.appName)
```

Reply:
I could, I just think this makes it more explicit what I'm trying to do.